### PR TITLE
[To rel/1.0][IOTDB-4433][IOTDB-4497] Add Cluster-Query-Command doc

### PR DESCRIPTION
**Pull request by CRZbulabula**
1. Move the Cluster-Query-Command contents from the Maintaince-Tool doc since they are Cluster SQLs but not executable tools.
2. Done some naming convention.

### CODE REVIEW

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 1]
```diff
@@ -0,0 +1,410 @@
...
+|       3|SchemaRegion|Running|root.sg2|          1|        0|         1| 127.0.0.1|   6667|Follower|
+|       3|SchemaRegion|Running|root.sg2|          1|        0|         2| 127.0.0.1|   6668|  Leader|
+|       3|SchemaRegion|Running|root.sg2|          1|        0|         3| 127.0.0.1|   6669|Follower|
++--------+------------+-------+--------+-----------+---------+----------+----------+-------+--------+
+Total line number = 6
+It costs 0.009s
+```
+
+### Region status definition
+Region inherits the status of the DataNode where the Region resides. And Region states are defined as follows
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:299]:
```suggestion
Region inherits the status of the DataNode where the Region resides in. And Region states are defined as follows
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 2]
```diff
@@ -0,0 +1,410 @@
...
+
+- **Running**: The DataNode where the Region resides is running properly, the Region can be read and written
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:301]:
```suggestion
- **Running**: The DataNode where the Region resides in is running properly, the Region can be read from and written to
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 3]
```diff
@@ -0,0 +1,410 @@
...
+- **Unknown**: The DataNode where the Region resides doesn't report heartbeat properly, the Region cannot be read or written
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:302]:
```suggestion
- **Unknown**: The DataNode where the Region resides in doesn't report heartbeat properly, the Region cannot be read from or written to
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 4]
```diff
@@ -0,0 +1,410 @@
...
+- **Removing**: The DataNode where the Region resides is being removed from the cluster, the Region cannot be read or written
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:303]:
```suggestion
- **Removing**: The DataNode where the Region resides in is being removed from the cluster, the Region cannot be read from or written to
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 5]
```diff
@@ -0,0 +1,410 @@
...
+- **ReadOnly**: The available disk space of the DataNode where the Region resides is lower than the disk_warning_threshold(5% by default). The Region can be read but can't be written and synchronize data
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:304]:
```suggestion
- **ReadOnly**: The available disk space of the DataNode where the Region resides in is lower than the disk_warning_threshold(5% by default). The Region can be read from but cannot be written to and synchronize data
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 6]
```diff
@@ -0,0 +1,410 @@
...
+|     5|  DataNode|Running|      127.0.0.1|        9005|                   | 127.0.0.1|   6669|            40012|              50012|   8779|
++------+----------+-------+---------------+------------+-------------------+----------+-------+-----------------+-------------------+-------+
+Total line number = 6
+It costs 0.340s
+```
+
+## Show Region information
+
+The cluster uses a SchemaRegion/DataRegion as a unit for schema/data replication and data management. 
+The Region status and distribution is helpful for system operation and maintenance testing, as shown in the following scenario:
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:166]:
```suggestion
The Region status and distribution is helpful for system operation and maintenance testing, as shown in the following scenarios:
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 7]
```diff
@@ -0,0 +1,410 @@
...
+The DataNode statuses are defined as follows:
+
+- **Running**: The DataNode is running properly and can be read and written
+- **Unknown**: The DataNode doesn't report heartbeat properly, the ConfigNode considers the DataNode as unreadable and un-writable
+- **Removing**: The DataNode is being removed from the cluster and cannot be read or written
+- **ReadOnly**: The remaining disk space of DataNode is lower than disk_warning_threshold(default is 5%), the DataNode can be read but can't be written or synchronize data
+
+## Show all Node information
+
+Currently, IoTDB supports show the information of all Nodes by the following SQL:
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:102]:
```suggestion
Currently, IoTDB supports showing the information of all Nodes by the following SQL:
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 8]
```diff
@@ -0,0 +1,410 @@
...
+### DataNode status definition
+The state machine of DataNode is shown in the figure below:
+<img style="width:100%; max-width:500px; max-height:500px; margin-left:auto; margin-right:auto; display:block;" src="https://github.com/apache/iotdb-bin-resources/blob/main/docs/UserGuide/Cluster/DataNode-StateMachine-EN.jpg?raw=true">
+
+The DataNode statuses are defined as follows:
+
+- **Running**: The DataNode is running properly and can be read and written
+- **Unknown**: The DataNode doesn't report heartbeat properly, the ConfigNode considers the DataNode as unreadable and un-writable
+- **Removing**: The DataNode is being removed from the cluster and cannot be read or written
+- **ReadOnly**: The remaining disk space of DataNode is lower than disk_warning_threshold(default is 5%), the DataNode can be read but can't be written or synchronize data
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:98]:
```suggestion
- **ReadOnly**: The remaining disk space of DataNode is lower than disk_warning_threshold(default is 5%), the DataNode can be read from but cannot be written to or synchronize data
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 9]
```diff
@@ -0,0 +1,410 @@
...
+
+### DataNode status definition
+The state machine of DataNode is shown in the figure below:
+<img style="width:100%; max-width:500px; max-height:500px; margin-left:auto; margin-right:auto; display:block;" src="https://github.com/apache/iotdb-bin-resources/blob/main/docs/UserGuide/Cluster/DataNode-StateMachine-EN.jpg?raw=true">
+
+The DataNode statuses are defined as follows:
+
+- **Running**: The DataNode is running properly and can be read and written
+- **Unknown**: The DataNode doesn't report heartbeat properly, the ConfigNode considers the DataNode as unreadable and un-writable
+- **Removing**: The DataNode is being removed from the cluster and cannot be read or written
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:97]:
```suggestion
- **Removing**: The DataNode is being removed from the cluster and cannot be read from or written to
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 10]
```diff
@@ -0,0 +1,410 @@
...
+It costs 0.006s
+```
+
+### DataNode status definition
+The state machine of DataNode is shown in the figure below:
+<img style="width:100%; max-width:500px; max-height:500px; margin-left:auto; margin-right:auto; display:block;" src="https://github.com/apache/iotdb-bin-resources/blob/main/docs/UserGuide/Cluster/DataNode-StateMachine-EN.jpg?raw=true">
+
+The DataNode statuses are defined as follows:
+
+- **Running**: The DataNode is running properly and can be read and written
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:95]:
```suggestion
- **Running**: The DataNode is running properly and is readable and writable
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 11]
```diff
@@ -0,0 +1,410 @@
...
+
+### ConfigNode status definition
+The ConfigNode statuses are defined as follows:
+
+- **Running**: The ConfigNode is running properly
+- **Unknown**: The ConfigNode doesn't report heartbeat properly
+
+## Show DataNode information
+
+Currently, IoTDB supports show DataNode information by the following SQL:
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:53]:
```suggestion
Currently, IoTDB supports showing DataNode information by the following SQL:
```

File: docs/UserGuide/Cluster/Cluster-Query-Command.md [hunk 12]
```diff
@@ -0,0 +1,410 @@
...
+    specific language governing permissions and limitations
+    under the License.
+
+-->
+
+# Cluster Information Query Command
+
+## Show ConfigNode information
+
+Currently, IoTDB supports show ConfigNode information by the following SQL:
```
Comment (Caideyipi) [inline docs/UserGuide/Cluster/Cluster-Query-Command.md:26]:
```suggestion
Currently, IoTDB supports showing ConfigNode information by the following SQL:
```

**Comment by qiaojialin** [comment]
https://github.com/apache/iotdb/pull/8286

**Pull request closed without merging**